UnitTests/UnitTests.FluentSharp.Ast/ExtensionMethods/Test_Ast_CSharp_ExtensionMethods_Code_Generation.cs

```diff
@@ -1,128 +1,128 @@
+                           .ResolveInvocationParametersType.assert_True();
+
+            //StaThread
+            compilerOptions.ExecuteInStaThread.assert_False();
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("StaThread"))
+                           .ExecuteInStaThread.assert_True();
+
+            //MtaThread
+            compilerOptions.ExecuteInMtaThread.assert_False();
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("MtaThread"))
+                           .ExecuteInMtaThread.assert_True();
+
+            //WorkOffline
+            compilerOptions.WorkOffline.assert_False();
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("WorkOffline"))
+                           .WorkOffline.assert_True();
+
+            //using
+            compilerOptions.Extra_Using_Statements.assert_Empty();
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("using ABC"))
+                           .Extra_Using_Statements.assert_Not_Empty()
+                                                  .first().assert_Is("ABC");
+            //O2Ref
+            compilerOptions.ReferencedAssemblies.assert_Not_Contains("ABC");
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("O2Ref:ABC"))
+                           .ReferencedAssemblies.assert_Contains("ABC");
+
+            //O2Download
+            compilerOptions.FilesToDownload.assert_Empty();
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("O2Download:ABC"))
+                           .FilesToDownload.assert_Not_Empty()
+                                                  .first().assert_Is("ABC");
+
+            //O2File
+            compilerOptions.ExtraSourceCodeFilesToCompile.assert_Empty();
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("O2File:ABC"))
+                           .ExtraSourceCodeFilesToCompile.assert_Not_Empty()
+                                                         .first().assert_Is("ABC");
+
+            
+            //O2Dir
+
+            var tempDir = "_DirToInclude".temp_Dir();
+            var file    = tempDir.folder_Create_File("tempFile.cs", "some code");
+            compilerOptions.ExtraSourceCodeFilesToCompile.assert_Not_Contains(file);
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("O2Dir:".append(tempDir)))
+                           .ExtraSourceCodeFilesToCompile.assert_Contains(file);
+            
+            tempDir.assert_Folder_Deleted();
+
+            //NuGet
+            compilerOptions.NuGet_References.assert_Empty();
+            compilerOptions.mapOptionsDefinedInsideComments(comments.add("NuGet:ABC"))
+                           .NuGet_References.assert_Not_Empty()
+                                            .first().assert_Is("ABC");
+        }
+    }
+}
```

Comment:
another case of file that was submitted with no changes (or at least I can't spot them)